packages/pie/src/Pie.tsx

```diff
@@ -169,6 +149,26 @@ const InnerPie = <RawDatum extends MayHaveLabel>({
         )
     }
 
+    if (enableArcLinkLabels && layers.includes('arcLinkLabels')) {
```

Comment:
Just reordering these sections so they line up with the order of the default `layers`. Looked like that was the intention here.